docs/package.json

```diff
@@ -16,7 +16,8 @@
     "postbuild:previews": "node --require esbuild-register ./scripts/generate-previews.ts",
     "postbuild:sitemap": "node --require esbuild-register ./scripts/generate-sitemap-robotstxt.ts",
     "start": "next start",
-    "test": "jest"
+    "test": "jest",
+    "test:links": "node --require esbuild-register ./scripts/link-checker-puppeteer.ts 0"
```

Comment:
do we need to pass 0 here or could we instead use default values in the script?

Reply:
Yea, we can make `0` as default, but since it's already here in the script, it doesn't make much difference. Also, the script can be a self-explained template on how to set the length.

Reply:
Would prefer to see `0` as the default value of the function where it is used. Without a flag indicating what `0` is, it is magic and difficult to understand

Reply:
+1

Reply:
Ok. I'll set `0` as default

Reply:
Done! 77db0129b802808925840a8fc92f3c192acfce35